integrations/lib/multiwoven/integrations/destination/http/config/spec.json

```diff
@@ -0,0 +1,27 @@
+{
+  "documentation_url": "https://docs.multiwoven.com/integrations/destination/http",
+  "stream_type": "dynamic",
+  "connection_specification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Http",
+    "type": "object",
+    "required": ["destination_url"],
+    "properties": {
+      "destination_url": {
+        "type": "string",
+        "title": "Destination url",
+        "order": 0
+      },
+      "Username": {
```

Comment:
It would be great if we add this inside `credentials` like the redshift as there is a high chance that we might need to accommodate various other authentication types like jwt tokens